lib/spark.js

```diff
@@ -263,17 +264,24 @@ export default class Spark {
 /**
  * @param {object} args
  * @param {number} args.roundLengthInMs
+ * @param {number} args.maxJitterInMs
  * @param {number} args.maxTasksPerRound
  * @param {number} args.lastTaskDurationInMs
  */
 export function calculateDelayBeforeNextTask ({
   roundLengthInMs,
+  maxJitterInMs,
```

Comment:
```suggestion
  maxJitterInMs = 0,
```

I think it's cleaner to add a default in the arguments directly, than in in the function body